test/disjointCoarseMallocPool.cpp

```diff
@@ -489,6 +491,12 @@ TEST_F(test, disjointCoarseMallocPool_random) {
                 void *ptr = umfPoolMalloc(pool, size);
                 ASSERT_NE(ptr, nullptr);
 
+                // check if it is empty and fill it with data
+                for (size_t k = 0; k < size; k++) {
+                    ASSERT_EQ(((unsigned char *)ptr)[k], 0);
```

Comment:
Wrong assumption: `umfPoolMalloc()` allocates size bytes of **uninitialized storage**, so it does not have to be zeroed.